packages/client/src/testing/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { MockedProvider } from "@apollo/react-testing"
+import { MockedProviderProps } from "@apollo/react-testing/lib/mocks/MockedProvider"
+import {
+  createHistory,
+  createMemorySource,
+  History,
+  LocationProvider
+} from "@reach/router"
+import * as enzyme from "enzyme"
+import * as React from "react"
+
+let mounts: enzyme.ReactWrapper[] = []
+
+afterEach(() => {
+  for (const m of mounts) {
+    m.unmount()
+  }
+  mounts = []
+})
+
+export function mount(
```

Comment:
This is a helper for rendering React components for tests. Most of our components will only work correctly if they are rendered inside of an Apollo provider component that handles making GraphQL queries. In this case `MockedProvider` sends back predefined responses instead of calling a real API. In addition `LocationProvider` tells @reach/router to use the provided mock history object instead of using `window.history`, which would attempt to use the real window location.

There is an `afterEach` callback here that automatically unmounts any mounted components after each test.